docs/README.md

```diff
@@ -15,7 +17,8 @@
 ## Introduction
 
 The Measurement Plug-In SDK for LabVIEW packages enable measurement developers
-to quickly create LabVIEW measurements and run them as a service. NI Measurement
+to quickly create LabVIEW measurements and run them as a service. They also enable
```

Comment:
Consider: "The packages provided with the Measurement Plug-in SDK enable measurement developers to quickly create LabVIEW measurements and run them as a service. They also allow developers to generate a LabVIEW client which communicates with the service over gRPC."